x-pack/plugins/security_solution/public/detections/components/osquery/osquery_flyout.tsx

```diff
@@ -19,11 +20,18 @@ const OsqueryActionWrapper = styled.div`
 
 export interface OsqueryFlyoutProps {
   agentId?: string;
-  defaultValues?: {};
+  defaultValues?: {
+    alertIds?: string[];
+    query?: string;
+    ecs_mapping?: { [key: string]: {} };
+    queryField?: boolean;
+  };
   onClose: () => void;
   ecsData?: Ecs;
 }
 
+const ACTIONS_QUERY_KEY = 'actions';
```

Comment:
add a comment to both places (`security_solution` and `osquery`) to make sure we keep the `queryKey` aligned